package.json

```diff
@@ -16,6 +16,10 @@
     "build": "webpack --mode production",
     "generate:stockInfo": "node database/dataGeneration/stockInfo/cluster.js",
     "generate:quoteInfo": "node database/dataGeneration/quoteInfo/cluster.js",
+    "seed:postgres": "psql -d stock_chart -a -f database/seed/schema.sql",
+    "preseed:cassandra": "cqlsh 127.0.0.1 9042 -f database/seed/schema.cql",
```

Comment:
Do you plan to keep these Cassandra seed scripts and the cql schema in your package? It may be cleaner to remove them in your final product, if you are only using Postgres to hold/manage your data.